gremlin-groovy/src/main/java/org/apache/tinkerpop/gremlin/groovy/jsr223/CompilationOptionsCustomizer.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.tinkerpop.gremlin.groovy.jsr223;
+
+import org.apache.tinkerpop.gremlin.jsr223.Customizer;
+
+/**
+ * Provides some custom compilation options to the {@link GremlinGroovyScriptEngine}.
+ *
+ * @author Stephen Mallette (http://stephen.genoprime.com)
+ */
+class CompilationOptionsCustomizer implements Customizer {
+
+    private final int expectedCompilationTime;
+
+    public CompilationOptionsCustomizer(final int expectedCompilationTime) {
```

Comment:
I believe `int` should be `Integer` looking at the other customizers.  However, there seems to be issues with customizers in general where in 3.2.5 they are deprecated and no longer public classes. So when I modify this class with Integer, (or try to use the newer, non 'customizer' package customizers) I get the following error.

```
      compilerCustomizerProviders: {
              "org.apache.tinkerpop.gremlin.groovy.jsr223.CompilationOptionsCustomizer":[1]}
```
Error:
```
[WARN] ScriptEngines - Could not instantiate CompilerCustomizerProvider implementation [org.apache.tinkerpop.gremlin.groovy.jsr223.CompilationOptionsCustomizer].  It will not be applied.
java.lang.IllegalAccessException: Class org.apache.tinkerpop.gremlin.groovy.engine.ScriptEngines can not access a member of class org.apache.tinkerpop.gremlin.groovy.jsr223.CompilationOptionsCustomizer with modifiers "public"
	at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
	at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(AccessibleObject.java:296)
	at java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:288)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:413)
	at org.apache.tinkerpop.gremlin.groovy.engine.ScriptEngines.lambda$createScriptEngine$15(ScriptEngines.java:416)
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:684)
	at org.apache.tinkerpop.gremlin.groovy.engine.ScriptEngines.createScriptEngine(ScriptEngines.java:399)
	at org.apache.tinkerpop.gremlin.groovy.engine.ScriptEngines.reload(ScriptEngines.java:187)
```